support-frontend/package.json

```diff
@@ -143,8 +143,8 @@
 		"@types/jsdom": "^21.1.6",
 		"@types/lodash.debounce": "^4.0.9",
 		"@types/node": "^22.10.1",
-		"@types/react": "18.0.3",
-		"@types/react-dom": "18.2.7",
+		"@types/react": "18.2.0",
```

Comment:
I was getting lots of linting errors with the older version. This matches the version of React we are actually using.